src/strategies/krm-blueprint.ts

```diff
@@ -54,9 +54,11 @@ export class KRMBlueprint extends BaseStrategy {
       this.path
     );
     for (const yamlPath of yamlPaths) {
-      const contents: GitHubFileContents = await this.github.getFileContents(
-        this.addPath(yamlPath)
-      );
+      const contents: GitHubFileContents =
+        await this.github.getFileContentsOnBranch(
```

Comment:
nit: I think it would be cleaner to just use the changes branch as a default but I don't feel strongly.

Separately, does this PR mean that it's very difficult / impossible to do the wrong thing? i.e. you can't accidentally fetch from the wrong branch?

Reply:
I hope so, I checked for everything I could see that seems to be "defaultBranch", "targetBranch", "baseBranch" that I may have missed. But release-please has been built with the assumption the default branch is both the branch PRs should target against and the branch from which PRs, so it's still possible we may be missing a few other places.